docs/source/federated-types/sharing-types.mdx

```diff
@@ -189,7 +189,74 @@ This definition works for querying Subgraph A, because Subgraph A's definition i
 
 > Note that Subgraph A's actual subgraph schema is _not_ modified. Within Subgraph A, `x` and `y` remain non-nullable.
 
-#### Omitting fields
+### Arguments
+
+Arguments for a shared field can differ between subgraphs in certain ways:
+
+* If an argument is _required_ in at least one subgraph, it can be _optional_ in other subgraphs. **It cannot be omitted.**
+* If an argument is _optional in every subgraph where it's defined_, it can be _omitted_ in other subgraphs.
```

Comment:
A big caveat that should added here for "always optional" arguments is that while they can be omitted in some subgraphs, they will _not_ be included in the supergraph. That is, an argument is included in the supergraph only if it appears in all subgraphs (that define the shareable field).

In other words, an optionally argument will automatically behave "as if" it was `@inaccessible` if it's not consistently defined in all subgraphs.

Reply:
Ooh that is a big one! I'll add.